src/quicklist.c

```diff
@@ -2003,7 +2003,7 @@ int quicklistTest(int argc, char *argv[], int accurate) {
             quicklistPushHead(ql, "hello", 6);
             quicklistEntry entry;
             quicklistIndex(ql, 0, &entry);
-            quicklistInsertAfter(ql, &entry, "abc", 4);
+            quicklistInsertBefore(ql, &entry, "abc", 4);
```

Comment:
I think we can add an assert to determine that the first element is "abc", and in the ```insert after 1 element`` test to determine that the last element is "abc".

Reply:
@sundb Sure do. Like this? Take a look for me, thanks